### ISSUE TITLE
Recording a route and pulling with range

### ISSUE THREAD

**Issue by Kuresu** [EasyFarm/EasyFarm#235]
Hi, first of all, excellent program. I'm really loving it.

But i noticed an issue, every now and then, if i wanna farm, i like making small routes to walk around in picking up mobs as i go.

Problem i encountered is, that when using Ranged attack to pull, the player will never stop to use Ranged attack, instead he will try to throw his weapon, and then keep running the recorded route, thus never actually standing still to properly pull.

If i'm lucky, he'll still try and engage after several tries of failed ranged attacks.

Is there any way to overrule the recorded route when doing ranged attack, so that player stands still, does his ranged attack, pulls and defeats mob, then continues on his routine?

**Comment by Mykezero** [comment]
Thanks for the heads up. I thought I had fixed that a long time ago; I'll take a look at what's going on.

**Comment by Kuresu** [comment]
That's awesome! Thank you!

I have one more request if possible.

Now that trusts is a thing, and since they're kinda your playable characters, could you, in the rest options of easyfarm, add in their characters as well? My healer Trust will eventually run out of mp, and because my main never stops, healing can't catch up in the long run.


Being able to rest for a min to get mp back up again could provide a more natural flow to things.

**Comment by Mykezero** [comment]
@Kuresu Absolutely. Trust support is something I've been wanting to implement for some time now. I hope I can get around to it some time soon!
